data/xml/J78.xml

```diff
@@ -0,0 +1,283 @@
+    The <fixed-case>TARGET</fixed-case> Project’s Interactive Computerized Multilingual Dictionary (John Burge)
+  </abstract>
+  <pages>1–61</pages>
+  <note>Microfiche 76</note>
+</paper>
+
+<paper id="1009">
+  <title>The <fixed-case>TARGET</fixed-case> <fixed-case>P</fixed-case>roject's Interactive Computerized Multilingual Dictionary</title>
+  <month>June</month>
+  <year>1978</year>
+  <pages>62–77</pages>
+  <author><first>John</first><last>Burge</last></author>
+  <note>Microfiche 76</note>
+</paper>
+
+<paper id="1010">
+  <title>The FINITE STRING, Volume 15, Number 4</title>
+  <month>December</month>
+  <year>1978</year>
+  <abstract>
+    <fixed-case>ACL</fixed-case>: Minutes Of the 16th Annual Business Meeting;
+    <fixed-case>ACL</fixed-case> Secretary-Treasurer’s Report;
+    <fixed-case>ACL</fixed-case> Officers For 1979;
+    <fixed-case>ACL</fixed-case> Officers 1963-1979;
+    <fixed-case>NSF</fixed-case>: Support for Computational Linguistics (Paul G. Chapin);
+    News: Short Notes;
+    News: <fixed-case>ARIST</fixed-case> Reprint Request (Martha E. Williams);
+    News: Summer Linguistics at <fixed-case>T</fixed-case>exas;
+    <fixed-case>P</fixed-case>h<fixed-case>D</fixed-case> Programs in Computational Linguistics;
+    Journal: Computational Linguistics and Computer Languages (T. Frey; T. Vamos);
+    Journal: Discourse Processes (Roy D. Freedle);
+    Book Notices (Mel’cuk R. Ravic);
+    <fixed-case>Y</fixed-case>ale <fixed-case>AI</fixed-case> Project Research Reports Available;
+    Summary of Research on Computational Aspects of Evolution Theories (Raymond D. Gumb);
+    Taxonomy: Information Sciences (Editors of Information Systems);
+    Machine Aids to Translation: A Concise State of the Art Bibliography (Wayne Zachary);
+    Reviews: On Human Communication, 3d Ed, by Colin Cherry (William L. Benzon);
+    Reviews: Abhångigkeitsgrammatik, by Jurgen Kunze (Kenneth F. Ballin);
+    Glancing, Referring and Explaining in the Dialogue System <fixed-case>HAM-RPM</fixed-case> (W. Wahlster; A. Jameson; W. Hoeppner);
+    A Critical Look at a Formal Model for Stratificational Linguistics (Alexander T. Borgida)
+  </abstract>
+  <pages>1–40</pages>
+  <note>Microfiche 77</note>
+</paper>
+
+<paper id="1011">
+  <title>Review: <i><fixed-case>O</fixed-case>n <fixed-case>H</fixed-case>uman <fixed-case>C</fixed-case>ommunication: <fixed-case>A</fixed-case> <fixed-case>R</fixed-case>eview, a <fixed-case>S</fixed-case>urvey, and a <fixed-case>C</fixed-case>riticism</i> 3rd ed., by <fixed-case>C</fixed-case>olin <fixed-case>C</fixed-case>herry</title>
+  <month>December</month>
+  <year>1978</year>
+  <pages>41–46</pages>
+  <author><first>William L.</first><last>Benzon</last></author>
+  <note>Microfiche 77</note>
+</paper>
+
+<paper id="1012">
+  <title>Review: <i><fixed-case>A</fixed-case>bhångigkeitsgrammatik</i>, by <fixed-case>J</fixed-case>urgen <fixed-case>K</fixed-case>unze</title>
```

Comment:
I think this should be `<fixed-case>A</fixed-case>bhängigkeitsgrammatik`.

Reply:
The table of contents really says å, but the paper itself has no accent. I'll change it to no accent.

Reply:
It looked German to me. A google search with å turns up nothing, but the search works with ä.

Reply:
[Amazon says `a`](https://www.amazon.com/Abhangigkeitsgrammatik-J-Kunze/dp/B00800HDMA).